test/unit/assets/octa-color.obj

```diff
@@ -0,0 +1,23 @@
+mtllib octa-color.mtl
```

Comment:
The plant obj is 9mb, which might slow down tests. Maybe for the missing material test, we can make a duplicate of this colored obj, but change the path from `octa-color.mtl` to something that doesn't exist?

Reply:
yes, that's what I had thought of too, made the same changes already for `plant.obj` and will push them asap.